manager/integration/tests/test_upgrade.py

```
@@ -63,156 +64,96 @@
 from common import v2_data_engine_cr_supported
 from common import get_instance_manager_names
 from common import LONGHORN_NAMESPACE
+from common import RETRY_COUNTS, RETRY_INTERVAL
 from test_orphan import create_orphaned_directories_on_host
 from test_orphan import delete_orphans
 from backupstore import set_backupstore_s3
 from backupstore import set_backupstore_nfs, mount_nfs_backupstore
 
+def wait_for_engine_image_upgraded(client, volume_name, engine_image_name):
```

Ruff (0.8.2)

Redefinition of unused `client` from line 38

(F811)